docs/Validation-and-Serialization.md

```diff
@@ -43,6 +43,8 @@ const schema = {
     required: ['x-foo']
   }
 }
+
+fastify.post('/the/url', { schema })
```

Comment:
`fastify.post('/the/url', { schema }, handler)` or `fastify.post('/the/url', { schema }, () => {})`

Reply:
Done